R/NAMESPACE

```diff
@@ -36,7 +36,8 @@ export(step_client)
 export(task_client)
 export(test)
 export(version_info)
+import(reticulate)
 importFrom(R6,R6Class)
 importFrom(magrittr,"%>%")
-importFrom(reticulate,import_builtins)
-importFrom(reticulate,py_get_attr)
+importFrom(rstudioapi,hasFun)
```

Comment:
Did we use `rstudioapi::hasFun` in this PR? Looks like I couldn't find the usage when searching across the repo.